### DISCUSSION TITLE
I'm getting rejected repeatedly

### DISCUSSION THREAD

**Discussion by Omar8345** [community/community#146918] [GitHub Education]
### Hi everyone,

Hi, I'm a student and its not my first time applying for Developer pack and I got accessed before, but I got a small issue,

First, the school I'm in (secondary/high school) isn't listed when I type it in, the only one listed is a branch in a city and my school is in a different city but in the same country, they both got the same name, and my school doesn't have a website.

I put a screenshot from my student portal which include

- Student Name
- Student Number
- National ID Number
- School Name
- Academic Year
- Grade Level
- Entry Date

And when it asked me to chose on the map (because I didn't select it I just wrote the name and didn't click the different branch), I just chose another institution literally next to my school and included in the notes when applying couple years ago the same way, different location on map but just wrote a note

The issue is, my application keeps getting denied how matter I tried, this is the reason mentioned besides using camera (which I think is not necessary if I'm putting a image of something digital)

> If your school provides an academic email, then please verify and use your academic email address. If your school does not provide an academic email address, then you must enable use of your device camera to capture your image.

I do have one, the issue is there is some policy in the emails where external emails we can't receive, that's the issue.

I'm not sure what to do, can anyone help me

**Comment by techy4shri** [comment]
Raise a support ticket on the support platform explaining your issue with all the details and your previous acceptance into the gihub education and developer pack, and the team will help you. While I don't know how your application got accepted before, I do know that the location and lack of an ID card can be an issue. Use your ID card and your email since your school doesn't provide one and you should be good to go.
Hope this helps :)

**Reply by Omar8345** [reply]
The email won't let me receive the link & and there is no such ID card for students. You know a link where I can contact the relevant team?

**Reply by techy4shri** [reply]
student ID card? How come an educational institute not have ID card for their students? Raise a ticket on https://support.github.com/request/education and explain your situation to them.

**Reply by Omar8345** [reply]
its just like any school KG-12, but I'm on high school, level 9 to 12

**Reply by techy4shri** [reply]
Well if you are a student then as long as you can provide an enrollment proof with a document showing that you're still studying there, it should work. Let's wait for the support's reply.

**Comment by Omar8345** [comment]
they haven't answered yet






---- On Fri, 13 Dec 2024 16:14:55 +0400 ***@***.*** wrote ----



hey so what did they reply on your support ticket?
